src/cljs/rems/actions/add_licenses.cljs

```diff
@@ -51,7 +51,8 @@
    (let [description (text :t.actions/add-licenses)]
      (post! "/api/applications/add-licenses"
             {:params {:application-id application-id
-                      :comment comment
+                      :comment (when comment
+                                 comment)
```

Comment:
isn't `(when comment comment)` the same as `comment`? Well ok, false maps to nil, but otherwise?

Reply:
Done, I decided to remove this change altogether, not needed for this patch.